### PR TITLE
Jlarkin/cleanup net5

### PR DESCRIPTION
**Pull request by j0shuams**

### CODE REVIEW

File: build/AzurePipelineTemplates/CsWinRT-Build-Steps.yml [hunk 1]
```diff
@@ -286,16 +286,16 @@ steps:
...
+  displayName: Stage Net6.0
   condition: and(succeeded(), eq(variables['BuildPlatform'], 'x86'), eq(variables['BuildConfiguration'], 'release'))
   inputs:
-    SourceFolder: $(Build.SourcesDirectory)\src\WinRT.Runtime\bin\$(BuildConfiguration)\net5.0
+    SourceFolder: $(Build.SourcesDirectory)\src\WinRT.Runtime\bin\$(BuildConfiguration)\net6.0
     Contents: |
       WinRT.Runtime.dll
       WinRT.Runtime.pdb
-    TargetFolder: $(Build.ArtifactStagingDirectory)\release_net5.0
+    TargetFolder: $(Build.ArtifactStagingDirectory)\release_net6.0
```
Comment (manodasanW) [inline build/AzurePipelineTemplates/CsWinRT-Build-Steps.yml:298]: duplicate of below.

File: nuget/Microsoft.Windows.CsWinRT.targets [hunk 2]
```diff
@@ -141,8 +141,7 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <CsWinRTInternalProjection Condition="'$(CsWinRTPrivateProjection)' == 'true'">-internal</CsWinRTInternalProjection>
       <CsWinRTEmbeddedProjection Condition="'$(CsWinRTEmbedded)' == 'true'">-embedded</CsWinRTEmbeddedProjection>
       <CsWinRTEmbeddedEnums Condition="'$(CsWinRTEmbeddedPublicEnums)' == 'true'">-public_enums</CsWinRTEmbeddedEnums>
-      <CsWinRTExeTFM Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) == 5">net5.0</CsWinRTExeTFM>
-      <CsWinRTExeTFM Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) > 5">net6.0</CsWinRTExeTFM>
+      <CsWinRTExeTFM Condition="$([MSBuild]::GetTargetFrameworkVersion('$(TargetFramework)')) == 6">net6.0</CsWinRTExeTFM>
```
Comment (manodasanW) [inline nuget/Microsoft.Windows.CsWinRT.targets:144]: > 6 to handle net7.

Comment (j0shuams) [reply nuget/Microsoft.Windows.CsWinRT.targets:144]: >= ?

Comment (manodasanW) [reply nuget/Microsoft.Windows.CsWinRT.targets:144]: Yes.

File: build/AzurePipelineTemplates/CsWinRT-Build-Steps.yml [hunk 3]
```diff
@@ -380,13 +369,13 @@ steps:
     PathtoPublish: $(Build.ArtifactStagingDirectory)\resx
     ArtifactName: ResX
 
-# Publish Net5.0
+# Publish Net6.0
```
Comment (manodasanW) [inline build/AzurePipelineTemplates/CsWinRT-Build-Steps.yml:372]: Same here, its also below.

File: build/AzurePipelineTemplates/CsWinRT-PublishToNuget-Stage.yml [hunk 4]
```diff
@@ -57,11 +57,11 @@ stages:
         downloadPath: $(Build.SourcesDirectory)
         extractTars: false
     
-# Download Net5.0
+# Download Net6.0
```
Comment (manodasanW) [inline build/AzurePipelineTemplates/CsWinRT-PublishToNuget-Stage.yml:60]: Same here

**Pull request closed without merging**